package.json

```diff
@@ -1,9 +1,10 @@
 {
   "packageManager": "yarn@2.4.3",
   "name": "svgo",
-  "version": "3.2.0",
+  "version": "3.3.0",
   "description": "Nodejs-based tool for optimizing SVG vector graphics files",
   "license": "MIT",
+  "type": "module",
```

Comment:
Just to verify, will this work with `require` from CJS packages? Like `const svgo = require('svgo');`.

I was under the impression that this change breaks CJS packages and only works with modules, but it's been some time since I last read the docs :)

Reply:
That change alone would break CJS packages, but I've updated our Rollup config to also create a CJS bundle, which we expose in our `package.json`.

See:

* https://github.com/svg/svgo/blob/main/rollup.config.js#L17
* https://github.com/svg/svgo/blob/main/package.json#L56
* https://github.com/svg/svgo/blob/main/test/svgo.cjs#L1

From my testing with `npm link`, this should work just fine with in CJS and `require`!

Reply:
Ah, nice, thanks for the explanation!